cfme/automate/dialog_element.py

```diff
@@ -142,15 +153,16 @@ def tree_path(self):
     def create(self, element_data=None):
         for element in element_data:
             view = navigate_to(self, "Add")
-            if(view.ele_label.value != ""):
+            if view.ele_label.value:
                 view.plus_btn.item_select("Add a new Element to this Box")
-            view.fill(element)
+            view.fill(element.get('element_information'))
+            if element.get('options') is not None:
```

Comment:
Again you should be able to call fill here, and when the value is None it will continue after filling nothing.

Reply:
Same as above , but removed these checks for 5.9 code 
cfme/automate/dialogs/dialog_element.py